content/src/content/jcr_root/apps/core/wcm/sandbox/components/teaser/v1/teaser/teaser.html

```diff
@@ -20,9 +20,11 @@
 
     <div class="cmp-teaser__image" data-sly-resource="${teaser.imageResource}"></div>
 
-    <h2 class="cmp-teaser__title" data-sly-test.title="${teaser.title}">${title}</h2>
-    <h3 class="cmp-teaser__description" data-sly-test.description="${teaser.description}">${description}</h3>
-
-    <a class="cmp-teaser__link" href="${teaser.linkURL}">${teaser.linkText}</a>
+    <h2 class="cmp-teaser__title" data-sly-test.title="${teaser.title}">
+        <a class="cmp-teaser__link" href="${teaser.linkURL}" data-sly-unwrap="${teaser.hideTitleLink}">${title}</a>
```

Comment:
The repeated use of `cmp-teaser__link` seems off in the context of BEM as it requires nesting for context (else it cannot be differentiated from the descriptions link). Would this be better named `cmp-teaster__title-link` ?

Reply:
Thanks for the suggestion, it makes a lot of sense